main.py

```diff
@@ -1,10 +1,37 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players:
+        data = json.load(players)
+
+    for nickname, abilities in data.items():
+        race, _ = Race.objects.get_or_create(
+            name=abilities["race"]["name"],
+            description=abilities["race"]["description"]
+        )
+        guild = None
+        if guild_ := abilities["guild"]:
+            guild, _ = Guild.objects.get_or_create(
+                name=guild_["name"],
+                description=guild_["description"],
+            )
+        for skill in abilities["race"]["skills"]:
+            Skill.objects.get_or_create(
+                name=skill["name"],
+                bonus=skill["bonus"],
+                race=race,
+            )
+        Player.objects.get_or_create(
```

Comment:
get_or_create operation can raise exceptions due to validation errors or database issues, so consider adding try/except blocks